crates/uv-settings/src/lib.rs

```diff
@@ -166,23 +167,12 @@ impl From<Options> for FilesystemOptions {
 
 /// Returns the path to the user configuration directory.
 ///
-/// This is similar to the `config_dir()` returned by the `dirs` crate, but it uses the
-/// `XDG_CONFIG_HOME` environment variable on both Linux _and_ macOS, rather than the
-/// `Application Support` directory on macOS.
+/// On Windows, use, e.g., C:\Users\Alice\AppData\Roaming
+/// On Linux and macOS, use `XDG_CONFIG_HOME` or $HOME/.config, e.g., /home/alice/.config.
 fn config_dir() -> Option<PathBuf> {
-    // On Windows, use, e.g., C:\Users\Alice\AppData\Roaming
-    #[cfg(windows)]
-    {
-        dirs_sys::known_folder_roaming_app_data()
-    }
-
-    // On Linux and macOS, use, e.g., /home/alice/.config.
-    #[cfg(not(windows))]
-    {
-        std::env::var_os("XDG_CONFIG_HOME")
-            .and_then(dirs_sys::is_absolute_path)
-            .or_else(|| dirs_sys::home_dir().map(|path| path.join(".config")))
-    }
+    etcetera::choose_base_strategy()
```

Comment:
I think this isn't quite the same. It looks like etcetera will use local `AppData` and then falls back to Roaming `AppData`, whereas the above always uses the Roaming version?

Reply:
`dirs-sys` uses the [`KnownFolderID`](https://learn.microsoft.com/en-us/windows/win32/shell/knownfolderid) `FOLDERID_RoamingAppData`, while `etcetera` uses the [`CSIDL`](https://learn.microsoft.com/en-us/windows/win32/shell/csidl) `CSIDL_APPDATA`. And `FOLDERID_RoamingAppData` is equivalent to `CSIDL_APPDATA` in CSIDL.

If the `SHGetFolderPathW` call fails, etcetera will fall back to use `self.home_dir.join("AppData").join("Roaming")`.  So it seems etcetera always uses the Roaming APPDATA too, they should be the same.